From: Bill MacAllister <bill@ca-zephyr.org>
Date: Fri, 15 May 2020 18:32:56 -0700
Subject: Fix location of kafs-dns

This patch modifies the kafs_dns.conf file delivered with the package
to match the location of the kafs-dns binary.  The location was changed
from up stream source to match Debian common usage.
---
 conf/kafs_dns.conf | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

Index: kafs-client/conf/kafs_dns.conf
===================================================================
--- kafs-client.orig/conf/kafs_dns.conf	2020-06-07 07:08:57.032191234 +0000
+++ kafs-client/conf/kafs_dns.conf	2020-06-07 07:08:57.020190679 +0000
@@ -1 +1 @@
-create  dns_resolver afsdb:*    *               /usr/libexec/kafs-dns %k
+create  dns_resolver afsdb:*    *               /usr/sbin/kafs-dns %k
